app.js

```diff
@@ -36,143 +40,89 @@ function cookiesSoldThisHour(customers, avgCookies) {
   return Math.floor(customers * avgCookies);
 }
 
-//function to populate array Data
-/*
-function cookiesSales(maxCustomers, minCustomers, avgCookies) {
-  var hourlySales = [];
+// hours of operation
+var hoursOpen = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+
+function CookieStore(name, minCustomers, maxCustomers, avgCookies, tagId){
+  this.name = name;
+  this.minCustomers = minCustomers;
+  this.maxCustomers = maxCustomers;
+  this.avgCookies = avgCookies;
+  this.hourlySales = [];
+  this.totalSales = 0;
+  this.tag = tagId;
+}
+
+CookieStore.prototype.populateCookieStore = function () {
   for (var iHours = 0; iHours < hoursOpen.length; iHours++) {
-    var customers = customersThisHour(maxCustomers, minCustomers);
-    var sales = cookiesSoldThisHour(customers, avgCookies);
-    stores[iStores].totalSales += sales;
-    console.log(hoursOpen[iHours] + ': ' + sales + ' cookies.');
-    createElement('li', 'class', 'hourly-sales', hoursOpen[iHours] + ': ' + sales + ' cookies.', elements[iStores]);
-    hourlySales.push(sales);
+    var customers = customersThisHour(this.maxCustomers, this.minCustomers);
+    var sales = cookiesSoldThisHour(customers, this.avgCookies);
+    this.totalSales += sales;
+//    console.log(this.totalSales);
+    this.hourlySales.push(sales);
   }
-  return hourlySales;
-}
-*/
+};
 
-// hours of operation
-var hoursOpen = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+function NewStore(name, minCustomers, maxCustomers, avgCookies, tagId){
+  var temp = new CookieStore(name, minCustomers, maxCustomers, avgCookies, tagId);
```

Comment:
consider a better variable name than just temp, even tempStore would make your code easier to understand